server/models/Collective.js

```diff
@@ -1433,25 +1441,27 @@ export default function(Sequelize, DataTypes) {
       CollectiveId: this.id,
     };
 
-    let isActive = false;
-    let approvedAt = null;
-    if (creatorUser.isAdmin) {
```

Comment:
I checked the code and found no valid reason to verify that this method exist, we always pass a valid user. It was introduced in https://github.com/opencollective/opencollective-api/commit/91d1f1974d605449a8383f4ae66af276e1038b40 to fix the tests, because we don't pass a `user` in the tests.

I'm gonna fix the tests, because adding this kind of checks is not a good practice and can be dangerous.